From 3d9dbac8d358ae9d13b72dbbc5986d8b8ce4d72d Mon Sep 17 00:00:00 2001
From: 0xsufi <0xsufi@noma.money>
Date: Sat, 9 Aug 2025 02:04:34 -0400
Subject: [PATCH] Add utreexod to valid node choices

---
 lncfg/chain.go | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/lncfg/chain.go b/lncfg/chain.go
index 3e6fdfdd8..adfa7377d 100644
--- a/lncfg/chain.go
+++ b/lncfg/chain.go
@@ -13,7 +13,7 @@ type Chain struct {
 	Active   bool   `long:"active" description:"DEPRECATED: If the chain should be active or not. This field is now ignored since only the Bitcoin chain is supported" hidden:"true"`
 	ChainDir string `long:"chaindir" description:"The directory to store the chain's data within."`
 
-	Node string `long:"node" description:"The blockchain interface to use." choice:"btcd" choice:"bitcoind" choice:"neutrino" choice:"nochainbackend"`
+	Node string `long:"node" description:"The blockchain interface to use." choice:"btcd" choice:"bitcoind" choice:"neutrino" choice:"nochainbackend" choice:"utreexod"`
 
 	MainNet         bool     `long:"mainnet" description:"Use the main network"`
 	TestNet3        bool     `long:"testnet" description:"Use the test network"`
-- 
2.43.0

